server/database/models/ForgotPassword.js

```diff
@@ -3,7 +3,7 @@ const mongoose = require('mongoose')
 const ForgotPasswordSchema = mongoose.Schema({
   code: String,
   email: String,
-  createdAt: { type: Date, default: Date.now, expires: '60' }
+  createdAt: { type: Date, default: Date.now, expires: '3600' }
```

Comment:
After doing some testing, I've found that expires is only working when set as `expires: '<num><denomination>'` with `<denomination>` being `h` or `m`. I couldn't in any way get seconds to work.. so I think try going with `expires: '1h'` (with the single quotes) and we'll be good

Reply:
I tried out and after 60 it was getting deleted...but I will change for that notation